### ISSUE TITLE
Custom time format for en-/decoding

### ISSUE THREAD

**Issue by AxelRHD** [BurntSushi/toml#408]
Can I use a custom time format for date en-/decoding? Maybe with a struct tag?

Can't find it in the documentation or the code.

Thanks!

**Comment by arp242** [comment]
No, the format is defined in the TOML spec; a custom format would break that.

You'll have to use a string – you can unmarshal that any way you want with a UnmarshalTOML method.
